micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSEdit.js

```
@@ -4,131 +1,192 @@
+    body: {},
+    config: { enabled: false },
+  });
+
+  const handleUpdate = async (formData, additionalProperties) => {
+    const transformedFormData = { ...formData };
+
+    // Prepare Localization Messages
+    const messages = [];
+    if (additionalProperties && typeof additionalProperties === "object") {
+      for (const fieldName in additionalProperties) {
+        if (additionalProperties.hasOwnProperty(fieldName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)